src/modules/workspaces/repositories/workspaces.repository.spec.ts

```diff
@@ -112,11 +112,9 @@ describe('SequelizeWorkspaceRepository', () => {
 
       const result = await repository.findWorkspaceUser({ memberId: '1' });
       expect(result).toBeInstanceOf(WorkspaceUser);
-      expect(result).toEqual(
-        expect.objectContaining({
-          ...workspaceUser.toJSON(),
-        }),
-      );
+      expect(result.toJSON()).toMatchObject({
+        ...workspaceUser.toJSON(),
+      });
```

Comment:
Not related to this PR but jest sometimes was not too happy with this test the way it was.